src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceContractTests.cs

```diff
@@ -19,25 +19,35 @@ public static class ServiceContractTests
     public static void DefaultSettings_Echo_RoundTrips_String_Buffered()
```

Comment:
Possibly, don't really want to get bogged down refactoring these tests, just some quick cleanup.